project/Dependencies.scala

```diff
@@ -2,7 +2,7 @@ import sbt._
 
 object Dependencies {
   private val akkaHttpCirceIntegrationV = "1.38.2"
-  private val akkaHttpV = "10.1.14" // scala-steward:off (CROM-6619)
+  private val akkaHttpV = "10.1.15"
```

Comment:
https://broadworkbench.atlassian.net/browse/CROM-6678 had this upgrading to 10.2.0. I get that you are just doing the upgrade necessary for the vulnerability, but I wonder if the scala steward off annotation and CROM-6619 reference should remain until it can be fully upgraded.

Out of curiosity, did you try upgrading to the most recent version?

Reply:
Can you leave the CROM-6619 comment so that when someone tries to upgrade akkaHttpV, they can reference/close that ticket? It does have a code change linked to it that could be helpful.

Reply:
sure will do.